lib/fathead/perl_doc/cover/functions.txt

```diff
@@ -0,0 +1,275 @@
+-A
```

Comment:
Yep, they're basically test load for the tests - no plan to remove them.

Is there an issue with this?

I'm updating fathead docs to detail the intent and functions of these files.